x/crosschain/keeper/v2_zevm_inbound.go

```diff
@@ -68,12 +67,12 @@ func (k Keeper) ProcessZEVMInboundV2(
 
 		// create inbound object depending on the event type
 		if withdrawalEvent != nil {
-			inbound, err = k.newWithdrawalInbound(ctx, from, txOrigin, foreignCoin, withdrawalEvent)
+			inbound, err = k.newWithdrawalInbound(ctx, txOrigin, foreignCoin, withdrawalEvent)
 			if err != nil {
 				return err
 			}
 		} else {
-			inbound, err = k.newCallInbound(ctx, from, txOrigin, foreignCoin, gatewayEvent)
+			inbound, err = k.newCallInbound(ctx, txOrigin, foreignCoin, gatewayEvent)
```

Comment:
Not required for this PR but consider refactoring to extract the common logic out of Withdraw and Call Inbound creation into a separate function .